ts/screens/services/ServiceDetailsScreen.tsx

```diff
@@ -341,16 +324,11 @@ class ServiceDetailsScreen extends React.Component<Props, State> {
           {metadata.description && <View spacer={true} large={true} />}
           {canRenderItems(this.state.isMarkdownLoaded, potServiceMetadata) && (
             <View>
-              {metadata.tos_url &&
-                renderInformationLinkRow(
-                  I18n.t("services.tosLink"),
-                  metadata.tos_url
-                )}
-              {metadata.privacy_url &&
-                renderInformationLinkRow(
-                  I18n.t("services.privacyLink"),
-                  metadata.privacy_url
-                )}
+              <TosAndPrivacyBox
```

Comment:
since both items could be undefined, we may show only the Heading block without children links, maybe we can add a check here to prevent component showing if neither `tos_url` nor `privacy_url` is available?

Reply:
Good point!

I'm divided on this, though: if we've got no links, we shouldn't call the component at all. Yet we _can_ 🤔 
I'll add the check but leave a note to the future reader because I think there is an opportunity for a better design.

Reply:
Perhaps to have a self-contained component and prevent the use without knowing this behavior, it might be worth move this check inside `TosAndPrivacy` in order to encapsulate the behavior, so that if both properties are undefined it won't be rendered. What you think?

Reply:
Yep, I find this weight more clear!

Reply:
Updated:
* [check](https://github.com/pagopa/io-app/pull/3200/commits/3ef427e3ae046af2cb180cc1a0fa231a59c5a767#diff-a6c4bb541039dee54bf32f6b1bb41ed1970b3b733f22099e3cb609b9f8c528b4R54) against edge condition
* [jsdoc](https://github.com/pagopa/io-app/pull/3200/commits/3ef427e3ae046af2cb180cc1a0fa231a59c5a767#diff-a6c4bb541039dee54bf32f6b1bb41ed1970b3b733f22099e3cb609b9f8c528b4R51) to inform _future self_ about this quirk
* added [a test](https://github.com/pagopa/io-app/pull/3200/commits/3ef427e3ae046af2cb180cc1a0fa231a59c5a767#diff-b61fe31b0909f945faba5f3797accd2d2ec9404e8d16f32f4adcadb1706e42d8R94) to ensure it stays there